src/Setup.elm

```diff
@@ -50,44 +51,34 @@ init =
 
 stringToFloat : String -> Float
 stringToFloat number =
-    let
-        maybeFloat =
-            String.toFloat number
-    in
-    case maybeFloat of
-        Nothing ->
-            0
-
-        Just value ->
-            value
+    withDefault 0 <| String.toFloat number
 
 
 yearsInput : Int -> Input.Label Msg -> (Float -> Msg) -> Element Msg
 yearsInput value label onChange =
-    el [ Element.centerX, Element.width <| px 345 ] <|
-        Input.slider
-            [ Element.height (Element.px 20)
-            , Element.behindContent
-                (Element.el
+    { onChange = onChange
+    , label = label
+    , min = 2007
+    , max = 2019
+    , step = Just 1
+    , value = toFloat value
+    , thumb =
+        Input.defaultThumb
+    }
+        |> Input.slider
+            [ Element.height <| Element.px 20
+            , Element.behindContent <|
+                Element.el
                     [ Element.width Element.fill
-                    , Element.height (Element.px 6)
-                    , Element.centerY
-                    , Background.color <| rgb255 0 0 200
+                    , Element.height <| Element.px 6
+                    , centerY
+                    , color <| rgb255 0 0 200
                     , Border.rounded 4
```

Comment:
If we have a `let` we can also use it for all those values that have no clear meaning and might change later.
For example, values like `20`, `6`, `4` and `rgb255 0 0 200`.